### DISCUSSION TITLE
为什么不能下载Quilt？？？

### DISCUSSION THREAD

**Discussion by starsmall123** [Meloong-Git/PCL#3552] [提问] [answered]
为什么不能下载Quilt？？？

**Comment by MoYuan-CN** [comment]
支持识别、安装 Quilt
请见 #48：等待 BMCLAPI 支持

**Comment by allMagicNB** [comment, marked as answer]
要等 BMCLAPI 那边的同步搞完，但是由于目前 Quilt 的 API 有 Bug，93哥反馈了但没人理他，现在就只能等 Quilt 官方修复
